app/views/environments/welcome.html.erb

```diff
@@ -11,4 +11,8 @@
   <div class="blank-slate-pf-main-action">
       <%= new_link(_('New Puppet environment'), {}, { :class => "btn-lg" }) %>
   </div>
+  <p><%= _('Or') %></p>
```

Comment:
That breaks https://www.patternfly.org/patterns/empty-state/, please try to adapt to use it @shlomizadok :smile: